Exiled.Permissions/Extensions/Permissions.cs

```diff
@@ -172,11 +172,18 @@ public static void Reload()
         /// <returns>Returns a value indicating whether the user has the permission or not.</returns>
         public static bool CheckPermission(this CommandSender sender, string permission)
         {
-            if (sender.FullPermissions || sender is ServerConsoleSender)
+            if (sender.FullPermissions || sender is ServerConsoleSender || sender is GameCore.ConsoleCommandSender || sender == Server.Host.Sender)
```

Comment:
```suggestion
            if (sender.FullPermissions || sender is ServerConsoleSender || sender == Server.Host.Sender)
```
`GameCore.ConsoleCommandSender` doesn't exist anymore.